addons/dragging/functions/fnc_startCarry.sqf

```diff
@@ -19,10 +19,12 @@
 params ["_unit", "_target"];
 TRACE_2("params",_unit,_target);
 
-// check weight
-private _weight = [_target] call FUNC(getWeight);
-
-if (_weight > missionNamespace getVariable ["ACE_maxWeightCarry", 1E11]) exitWith {
+// exempt from weight check if object has override variable set
+if (!GETVAR(_target,GVAR(ignoreWeightCarry),false) && {
+    private _weight = ([_target] call FUNC(getWeight));
+    _weight > (GETMVAR(ACE_maxWeightCarry,1E11))
+    }) exitWith {
```

Comment:
```suggestion
    private _weight = [_target] call FUNC(getWeight);
    _weight > GETMVAR(ACE_maxWeightCarry,1E11)
}) exitWith {
```
formatting. Also testing GitHub `suggestion`.

Reply:
Hmm, replaced only 1 line with multiple

Reply:
Yeah, apparently you can only replace 1 line as of now. :(